runtimes/common/src/benchmark/mod.rs

```diff
@@ -0,0 +1,33 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use parity_scale_codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use sp_std::vec::Vec;
+
+pub mod treasury;
+pub mod xcm;
+
+#[derive(Clone, Copy, Default, Debug, Encode, Decode, PartialEq, Eq, TypeInfo)]
+pub struct DummySignature;
+
+impl<A> From<(A, Vec<u8>)> for DummySignature {
```

Comment:
Uh? What is this? Can you explain?

Reply:
Comments are added here: [8cf36fd](https://github.com/KILTprotocol/kilt-node/pull/655/commits/8cf36fda91c8cc9143d3cc1811d7856fcea47ac5)